src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    height: 134px;
+    margin: 0 auto 40px;
+    max-width: 160px;
+    display: block;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color;
+
+    @include transition-mixin;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__code {
+    margin: 0;
+    padding: 0;
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color;
```

Comment:
Ensure that the variable `$color` is defined in your SCSS files. If it's not, this will cause a compilation error.